### PR TITLE
Type of book tab

### PR DESCRIPTION
**Pull request by miri**
#59

### CODE REVIEW

**Comment by miri** [comment]
image

File: haskala/modules/custom/haskala_book/haskala_book.module [hunk 1]
```diff
@@ -23,24 +24,33 @@ function haskala_book_ctools_plugin_directory($module, $plugin) {
...
+
     $query->fieldCondition('field_producer', 'target_id', $person_nid);
   }
+  if ($role) {
+    $term = taxonomy_get_term_by_name($role);
+    $role_tid = key($term);
+    $query->fieldCondition('field_role', 'tid', $role_tid);
+  }
   $result = $query->execute();
+  dpm($result);
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/haskala_book.module]: Remove dpm

File: haskala/modules/custom/haskala_book/haskala_book.module [hunk 2]
```diff
@@ -112,6 +122,20 @@ function haskala_book_get_tab_fields($wrapper, $tab, $node_view, $book_nid) {
       $field = render($node_view[$field_name]);
       $tab_fields_main[] = $field;
 
+      // Add translator name to Type of book tab, if book is a translation.
+      if ($field_name == 'field_book') {
+        $production_node = haskala_book_get_production($book_nid, NULL, t('Translator'));
+        if ($production_node) {
+          //$production_wrapper = entity_metadata_wrapper('node', $production_node);
```
Comment (bitamar) [inline haskala/modules/custom/haskala_book/haskala_book.module]: Remove comment

**Comment by miri** [comment]
Merged.
Waiting for Marko answer about:
https://github.com/Gizra/haskala/issues/59#issuecomment-72806861

**Pull request closed without merging**